packages/cactus-plugin-satp-hermes/src/main/typescript/api1/admin/healthcheck-endpoint.ts

```diff
@@ -34,6 +36,24 @@ export class HealthCheckEndpointV1 implements IWebServiceEndpoint {
     const level = this.options.logLevel || "INFO";
     const label = this.className;
     this.log = LoggerProvider.getOrCreate({ level, label });
+    this.monitorService = MonitorService.createOrGetMonitorService({
+      logLevel: options.logLevel,
+    });
+    void this.initializeMonitorService();
+  }
+
+  private async initializeMonitorService(): Promise<void> {
+    const fnTag = `${this.className}#initializeMonitorService()`;
+    try {
+      await this.monitorService.init();
```

Comment:
this functionality should be part of the monitoring service. In particular public methods that can be used throughout the code